src/Visualization.jl

```diff
@@ -1,28 +1,73 @@
-using Luxor
-using MathTeXEngine
-using LaTeXStrings
+using Cobweb: h
 
-texname(::Type{Op}) where {Op<:Operator} = LaTeXString(String(nameof(Op)))
+texname(::Type{Op}) where {Op<:Operator} = String(nameof(Op))
 
-texname(::Type{Sd}) = L"S^\dagger"
-texname(::Type{Td}) = L"T^\dagger"
+texname(::Type{Sd}) = """S<tspan font-size="60%" baseline-shift="super">†</tspan>"""
+texname(::Type{Td}) = """T<tspan font-size="60%" baseline-shift="super">†</tspan>"""
 
-texname(::Type{Rx}) = L"R_X"
-texname(::Type{Ry}) = L"R_Y"
-texname(::Type{Rz}) = L"R_Z"
+texname(::Type{Rx}) = """R<tspan font-size="60%" baseline-shift="sub">X</tspan>"""
+texname(::Type{Ry}) = """R<tspan font-size="60%" baseline-shift="sub">Y</tspan>"""
+texname(::Type{Rz}) = """R<tspan font-size="60%" baseline-shift="sub">Z</tspan>"""
 
-texname(::Type{Hz}) = L"H_Z"
+texname(::Type{Hz}) = """H<tspan font-size="60%" baseline-shift="sub">Z</tspan>"""
 
-texname(::Type{FSim}) = L"F_S"
+texname(::Type{FSim}) = """F<tspan font-size="60%" baseline-shift="sub">S</tspan>"""
 
 function draw end
 export draw
 
+const DRAWING_STYLE = h.style(
+    """
+    .wire {
+        stroke: black;
+        stroke-width: 2px;
+    }
+
+    .lane {}
+    .virtual {}
+
+    .block {
+        stroke: black;
+        fill: white;
```

Comment:
Maybe make the background of the blocks `"transparent"` so it has the same background as the other lanes and wires?
```suggestion
        fill: transparent;
```